packages/elements/src/components/styles/mdc-customize.scss

```diff
@@ -0,0 +1,20 @@
+@use 'colors';
+@use 'fonts';
+
+// @material/theme
+@use '@material/theme' with (
+  $primary: colors.ino-color(primary),
+  $secondary: colors.ino-color(secondary),
+  $on-primary: colors.ino-color(primary, contrast),
+  $on-secondary: colors.ino-color(secondary, contrast),
+);
+
+// @material/typography
+@use '@material/typography' with (
+  $font-family: fonts.$font-family-sans-serif
+);
+
+@forward '@material/typography' as mdc-typo-*;
```

Comment:
If we want to use any exports of mdc typography or theme, we import it from `mdc-customize` where it's initialized with our branding. But because both `@material/typography` and `@material/theme` export a `coreStyles` mixin, it throws an error. The `as ...*` prefixes every export so we can e.g. use the `base` mixin of `typography` this way: `@include mdc-typo-base()`.

We use the `@material/typography` in our storybook.